{{cookiecutter.project_slug}}/Makefile

```diff
@@ -116,14 +159,18 @@ clean-pyc: ## remove Python file artifacts
 	find . -name '*~' -exec rm -f {} +
 	find . -name '__pycache__' -exec rm -fr {} +
 
-.PHONY: clean-test
-clean-test: ## remove test and coverage artifacts
-	rm -fr .tox/
+.PHONY: clean-coverage
+clean-coverage: ## remove coverage artifacts
 	rm -f .coverage
+	rm -f .coverage.*
 	rm -fr htmlcov/
+
+.PHONY: clean-test
+clean-test: ## remove test artifacts
+	rm -fr .tox/
 	rm -fr .pytest_cache
 
 .PHONY: clean-docs
 clean-docs: ## remove previously built docs
 	rm -f docs/api/*.rst
-	-$(MAKE) -C docs clean 2>/dev/null  # this fails if sphinx is not yet installed
+	$(MAKE) -C docs clean 2>/dev/null  # this fails if sphinx is not yet installed
```

Comment:
please revert this change

the situation is that `make install` has a dependency on the `clean` target which uses `sphinx` to clean the docs. But sphinx may not be installed yet in a fresh env, so the command would fail. Adding a `-` in front of a command allows failures.

Reply:
Sure, I must have skipped it while merging with the other changes.